test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,8 +34,33 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+
+
     }
 
+
+    @Test
+    public void isAnyNull() throws Exception {
+        // some null objects
+        assertAreNull("abc", null);
+        assertAreNull(null, null, null);
+
+        // no null objects
+        assertAreNotNull("abc", "abc");
+        assertAreNotNull('b', "abc");
+        assertAreNotNull("", "");
+
```

Comment:
Good job on attempting this LO!

It's good that you separated your tests into 2 categories. As mentioned in tutorial, it might also be a good idea to test a mixture of different data types.